source/extensions/filters/http/ext_authz/ext_authz.cc

```diff
@@ -138,6 +138,7 @@ void Filter::onComplete(Filters::Common::ExtAuthz::ResponsePtr&& response) {
   ASSERT(cluster_);
   state_ = State::Complete;
   using Filters::Common::ExtAuthz::CheckStatus;
+  Stats::StatName empty_string;
```

Comment:
Would `empty_stat` make more sense as a name, since this isn't a string? (I read its use below before noticing the declaration site and was confused.)

Reply:
Changed to empty_stat_name.